packages/token-balance-cache/src/providers/ethToken.ts

```diff
@@ -1,17 +1,13 @@
-import { providers } from 'ethers';
-import type { ERC20, ERC721 } from 'common-common/src/eth/types';
-import { ERC20__factory, ERC721__factory } from 'common-common/src/eth/types';
-
-import type { IChainNode } from '../types';
+import type { EthBPOpts, IChainNode } from '../types';
 import { BalanceProvider } from '../types';
 import { BalanceType } from 'common-common/src/types';
+import Web3 from 'web3';
+import type { WebsocketProvider } from 'web3-core';
 
-type EthBPOpts = {
-  tokenAddress?: string;
-  contractType?: string;
-};
-
-export default class EthTokenBalanceProvider extends BalanceProvider<EthBPOpts> {
+export default class EthTokenBalanceProvider extends BalanceProvider<
+  Web3,
+  EthBPOpts
```

Comment:
This provider no longer needs options, since it's only for native token.

Reply:
yes!